codice/ddf
# [2.25.x] DDF-6262 Fixes the issue of property replacement not working when creating a source

#### What does this PR do?
Fixes the issue of property replacement not working when creating a source

#### Who is reviewing it? 
@bakejeyner 
@emmberk
@garrettfreibott
@stustison

#### How should this be tested?
- Create a CSW and Open Search source with system properties in the URL
- Make sure these sources are available
- Verify that querying from these sources succeeds

#### What are the relevant tickets?
#6262 

#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
